src/concurrency/async/async-await.md

```diff
@@ -10,7 +10,7 @@ At a high level, async Rust code looks very much like "normal" sequential code:
 use futures::executor::block_on;
 
 async fn count_to(count: i32) {
-    for i in 1..=count {
+    for i in 0..count {
```

Comment:
We will get an extra line of output now (here and everywhere else). You could consider reducing the input `count` to combat this.

I remember having changed some inputs from, say, 5 or 10 to just 3 when teaching precisely so that I can have everything on screen and reduce the vertical scrolling.

Reply:
I think there are few enough lines in most of these examples that this is reasonable.